src/content/guides/production.md

```diff
@@ -148,42 +148,6 @@ __package.json__
 Feel free to run those scripts and see how the output changes as we continue adding to our _production_ configuration.
 
 
-## Minification
-
-Note that while the [`UglifyJSPlugin`](/plugins/uglifyjs-webpack-plugin) is a great place to start for minification, there are other options out there. Here are a few more popular ones:
-
-- [`BabelMinifyWebpackPlugin`](https://github.com/webpack-contrib/babel-minify-webpack-plugin)
-- [`ClosureCompilerPlugin`](https://github.com/roman01la/webpack-closure-compiler)
-
-If you decide to try another minification plugin, just make sure your new choice also drops dead code as described in the [tree shaking](/guides/tree-shaking) guide.
-
-
-## Source Mapping
-
-We encourage you to have source maps enabled in production, as they are useful for debugging as well as running benchmark tests. That said, you should choose one with a fairly quick build speed that's recommended for production use (see [`devtool`](/configuration/devtool)). For this guide, we'll use the `source-map` option in _production_ as opposed to the `inline-source-map` we used in _development_:
-
-__webpack.prod.js__
-
-``` diff
-  const merge = require('webpack-merge');
-  const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
-  const common = require('./webpack.common.js');
-
-  module.exports = merge(common, {
-    mode: 'production',
-+   devtool: 'source-map',
-    plugins: [
--     new UglifyJSPlugin()
-+     new UglifyJSPlugin({
-+       sourceMap: true
-+     })
-    ]
-  });
-```
-
-T> Avoid `inline-***` and `eval-***` use in production as they can increase bundle size and reduce the overall performance.
-
-
 ## Specify the Mode
```

Comment:
Mode is now higher, as it is set key to both next paragraphs, and was coming after them in prev structure.

Reply:
I see, thanks